cabal-helper.cabal

```diff
@@ -55,16 +55,21 @@ library
                        CabalHelper.Shared.Sandbox
                        Paths_cabal_helper
   ghc-options:         -Wall
-  build-depends:       base          < 5    && >= 4.5
-                     , Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
+  build-depends:       Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
                      , directory     < 1.4  && >= 1.1.0.2
                      , filepath      < 1.5  && >= 1.3.0.0
                      , transformers  < 0.6  && >= 0.3.0.0
                      , mtl           < 2.3  && >= 2.0
                      , process       < 1.7  && >= 1.1.0.1
-                     , unix          < 2.8  && >= 2.5.1.1
                      , semigroupoids < 5.3  && >= 5.2.1
                      , ghc-prim
+  if os(windows)
+    build-depends:     base          < 5    && >= 4.7
+                     , unix-compat
+  else
+    build-depends:     base          < 5    && >= 4.5
```

Comment:
I'd prefer something like:

```
build-depends:     base < 5 && >= 4.5
  if os(windows)
    build-depends: base        >= 4.7
build-depends:     Cabal < 1.2.3.4 && ...
                   ...
```

Less duplication of information that way and for ease of editing and reading the `if` block should come right after main version bound.